### PR TITLE
Add service.type and service.loadBalancerIP for kiali service

### PR DESCRIPTION
**Pull request by FengyunPan2**
Current users login kiali dashboard by ingress.
After adding this pr, we can login it by NodePort service or
loadBalancerIP.

[x] Installation

### CODE REVIEW

File: install/kubernetes/helm/istio/charts/kiali/templates/service.yaml [hunk 1]
```diff
@@ -3,15 +3,30 @@ kind: Service
...
-  - name: http-kiali
+  - name: {{ .Values.service.name }}
+    targetPort: 20001
     protocol: TCP
-    port: 20001
+    port: {{ .Values.service.externalPort }}
   selector:
     app: kiali
+{{- if .Values.service.loadBalancerIP }}
+  loadBalancerIP: "{{ .Values.service.loadBalancerIP }}"
```
Comment (waret) [inline install/kubernetes/helm/istio/charts/kiali/templates/service.yaml]: As your reference, `type: LoadBalancer` should be added here in my test with metallb.

**Comment by waret** [comment]
I think prs about install/kubernetes/helm/* maybe commited to https://github.com/istio/installer first. In addition, prometheus/templates/service.yaml and tracing/templates/service.yaml maybe need service.loadBalancerIP too.

**Review by morvencao — COMMENTED** [review]
Please raise PR to new installer: https://github.com/istio/installer

**Comment by FengyunPan2** [comment]
@waret @morvencao I have sended a pr to istio installer: https://github.com/istio/installer/pull/397

**Comment by howardjohn** [comment]
I don't think we have the bandwidth to support making changes and testing those changes to the old installer for Kiali. We should merge this in for istio/installer so it can be used for the operator. Thanks for sending the change there!

**Pull request closed without merging**